ui/chat/src/discussion.ts

```diff
@@ -84,11 +84,18 @@ function renderInput(ctrl: Ctrl): VNode | undefined {
 let mouchListener: EventListener;
 
 const setupHooks = (ctrl: Ctrl, chatEl: HTMLElement) => {
-  chatEl.addEventListener('keypress',
+  chatEl.addEventListener('keydown',
```

Comment:
why switch from `keypress` to `keydown`?

Reply:
I read [https://developer.mozilla.org/en-US/docs/Web/API/Document/keypress_event](url) that key press was depreciated and was having trouble triggering it as an event, and `keydown` worked. 

I'll switch it back though, as what you're saying about session storage makes much more sense then what I'm doing here.

Reply:
if `keypress` is deprecated then it's good to stop using it! but then you have to worry about having broken something... best to just leave it when new to someone else's codebase is my attitude